ci-operator/step-registry/ipi/aws/pre/workers-rhel7/provision/ipi-aws-pre-workers-rhel7-provision-commands.sh

```diff
@@ -0,0 +1,211 @@
+#!/bin/bash
```

Comment:
I think the pre directory is only a collection of chain and doesn't really have actual commands. Can we move this to ipi/install/workers-rhel7 or someplace else ?

Reply:
Since this step is AWS specific I was trying to find a place in an `aws/` path.  I can move it to ipi/install/workers-rhel7 and just name it `aws-provision` if that is a better approach.

Reply:
I would keep this in `ipi/install/workers-rhel7/aws/provision` if we want to keep aws in the name.

Reply:
Updated.